experimenter/experimenter/slack/notification.py

```diff
@@ -15,18 +15,39 @@ def _get_slack_client():
     return WebClient(token=settings.SLACK_AUTH_TOKEN)
 
 
-def _get_user_mentions(client, emails):
+def _lookup_users(client, emails):
     mentions = []
+    user_ids = []
     for email in emails:
         if not email:
             continue
         try:
             response = client.users_lookupByEmail(email=email)
-            mentions.append(f"<@{response['user']['id']}>")
+            user_id = response["user"]["id"]
+            mentions.append(f"<@{user_id}>")
+            user_ids.append(user_id)
         except SlackApiError as e:
             logger.warning(f"Could not find Slack user for {email}: {e}")
             continue
-    return " ".join(mentions)
+    return " ".join(mentions), user_ids
```

Comment:
I mean I'd probably keep this function just to lookup and return the user ids and move constructing the mentions elsewhere rather than fold them both into one func